Repository/RevisionRepository.php

```diff
@@ -458,7 +458,7 @@ public function lockRevision($revisionId, $username, \DateTime $lockUntil)
      * @param \DateTime $now
      * @return mixed
      */
-    public function finaliseRevision(ContentType $contentType, $ouuid, \DateTime $now)
+    public function finaliseRevision(ContentType $contentType, $ouuid, \DateTime $now, string $lockUser = 'SYSTEM_MIGRATE')
```

Comment:
I don't like the default user. The lockUser should always be provided.
```suggestion
    public function finaliseRevision(ContentType $contentType, $ouuid, \DateTime $now, string $lockUser)
```

Reply:
a19f9458a3ffedd3f108acc0e78aaa46debc98a0